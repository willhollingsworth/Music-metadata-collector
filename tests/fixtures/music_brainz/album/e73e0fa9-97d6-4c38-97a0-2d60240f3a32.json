{
  "disambiguation": "",
  "media": [
    {
      "position": 1,
      "format": "CD",
      "tracks": [
        {
          "recording": {
            "disambiguation": "",
            "video": false,
            "title": "Aurora",
            "length": 421213,
            "artist-credit": [
              {
                "name": "Teho",
                "joinphrase": "",
                "artist": {
                  "name": "Teho",
                  "sort-name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "country": "FR",
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
                }
              }
            ],
            "first-release-date": "2022-03-25",
            "id": "67caeda7-e9a5-4e5a-828e-8d9ae46e9d21"
          },
          "position": 1,
          "title": "Aurora",
          "length": 421213,
          "number": "1",
          "artist-credit": [
            {
              "artist": {
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "name": "Teho",
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person"
              },
              "name": "Teho",
              "joinphrase": ""
            }
          ],
          "id": "f3e493b3-ceb6-4559-8064-58024a400104"
        },
        {
          "title": "At Any Cost",
          "number": "2",
          "artist-credit": [
            {
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "sort-name": "Teho",
                "name": "Teho"
              },
              "joinphrase": "",
              "name": "Teho"
            }
          ],
          "id": "7745b669-e722-4ee5-9758-0d1c1257ff5e",
          "length": 273000,
          "position": 2,
          "recording": {
            "disambiguation": "",
            "video": false,
            "title": "At Any Cost",
            "length": 273000,
            "id": "dda24014-a791-4e82-b256-32b2b32d2064",
            "artist-credit": [
              {
                "artist": {
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df"
                },
                "joinphrase": "",
                "name": "Teho"
              }
            ],
            "first-release-date": "2022-03-25"
          }
        },
        {
          "length": 355746,
          "id": "0deadec4-3360-4b3b-9f65-5abce9582f36",
          "number": "3",
          "artist-credit": [
            {
              "joinphrase": "",
              "name": "Teho",
              "artist": {
                "type": "Person",
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "sort-name": "Teho",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df"
              }
            }
          ],
          "title": "Millie",
          "recording": {
            "video": false,
            "disambiguation": "",
            "title": "Millie",
            "first-release-date": "2022-03-25",
            "artist-credit": [
              {
                "joinphrase": "",
                "name": "Teho",
                "artist": {
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "country": "FR",
                  "name": "Teho",
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
                }
              }
            ],
            "id": "71b72fb4-acbc-4c4d-9c03-4ec4aba198ad",
            "length": 355746
          },
          "position": 3
        },
        {
          "title": "Stern",
          "number": "4",
          "artist-credit": [
            {
              "artist": {
                "name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "sort-name": "Teho",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "type": "Person",
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
              },
              "name": "Teho",
              "joinphrase": ""
            }
          ],
          "id": "4bc571dc-7b00-4f35-b9ff-7229b9eaaab3",
          "length": 379360,
          "position": 4,
          "recording": {
            "length": 379360,
            "first-release-date": "2022-03-25",
            "artist-credit": [
              {
                "artist": {
                  "name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
                },
                "name": "Teho",
                "joinphrase": " & "
              },
              {
                "artist": {
                  "name": "Ceas",
                  "country": null,
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "id": "671d8f99-1eba-4c31-b727-e4a93513a97e",
                  "sort-name": "Ceas",
                  "disambiguation": "Antoine Rigail",
                  "type": "Person"
                },
                "joinphrase": "",
                "name": "Ceas"
              }
            ],
            "id": "fdf70e97-4645-4d03-8c65-91660a0b4be9",
            "title": "Stern",
            "disambiguation": "",
            "video": false
          }
        },
        {
          "position": 5,
          "recording": {
            "id": "eb2ef387-df87-4bcf-9af8-87b206b11996",
            "artist-credit": [
              {
                "artist": {
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "type": "Person",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "sort-name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "name": "Teho"
                },
                "name": "Teho",
                "joinphrase": " & "
              },
              {
                "artist": {
                  "type": "Person",
                  "disambiguation": "Aloys Almon",
                  "sort-name": "Aalson",
                  "id": "c8a43b08-9a5f-4177-891f-85486554dbc1",
                  "country": null,
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "name": "Aalson"
                },
                "name": "Aalson",
                "joinphrase": ""
              }
            ],
            "first-release-date": "2022-03-25",
            "length": 324000,
            "title": "Beyond",
            "video": false,
            "disambiguation": ""
          },
          "number": "5",
          "artist-credit": [
            {
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "name": "Teho",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880"
              },
              "joinphrase": "",
              "name": "Teho"
            }
          ],
          "id": "7eebf6fc-2061-4a6d-8483-ddeeddf27407",
          "length": 324000,
          "title": "Beyond"
        },
        {
          "recording": {
            "title": "Unlocked",
            "id": "ac4070ff-2191-415b-9532-68c7dd2f9c3f",
            "first-release-date": "2022-03-25",
            "artist-credit": [
              {
                "artist": {
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "type": "Person",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "name": "Teho"
                },
                "name": "Teho",
                "joinphrase": ""
              }
            ],
            "length": 424973,
            "video": false,
            "disambiguation": ""
          },
          "position": 6,
          "title": "Unlocked",
          "length": 424973,
          "artist-credit": [
            {
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "name": "Teho",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880"
              },
              "joinphrase": "",
              "name": "Teho"
            }
          ],
          "number": "6",
          "id": "b11a19fe-8002-4eba-9daf-e18375624ad0"
        },
        {
          "length": 333506,
          "number": "7",
          "artist-credit": [
            {
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "name": "Teho"
              },
              "name": "Teho",
              "joinphrase": ""
            }
          ],
          "id": "4a11e843-77e9-471c-b766-25243a1caff8",
          "title": "Fury",
          "recording": {
            "title": "Fury",
            "artist-credit": [
              {
                "artist": {
                  "name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
                },
                "name": "Teho",
                "joinphrase": " & "
              },
              {
                "name": "Joris Delacroix",
                "joinphrase": "",
                "artist": {
                  "id": "f2c454ec-69a2-49a7-a79c-eaabef25ba44",
                  "sort-name": "Delacroix, Joris",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "name": "Joris Delacroix",
                  "type": "Person",
                  "disambiguation": "French house DJ & producer"
                }
              }
            ],
            "first-release-date": "2022-03-25",
            "id": "2c898955-4305-413d-8548-64ee75aaf06f",
            "length": 333506,
            "video": false,
            "disambiguation": ""
          },
          "position": 7
        },
        {
          "length": 323226,
          "number": "8",
          "artist-credit": [
            {
              "artist": {
                "name": "Teho",
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "type": "Person",
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
              },
              "name": "Teho",
              "joinphrase": ""
            }
          ],
          "id": "2ef4bd37-5db9-4474-a37c-decc71860185",
          "title": "Dark Matter",
          "recording": {
            "disambiguation": "",
            "video": false,
            "length": 323226,
            "artist-credit": [
              {
                "artist": {
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "name": "Teho",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "type": "Person"
                },
                "joinphrase": "",
                "name": "Teho"
              }
            ],
            "first-release-date": "2022-03-25",
            "id": "2bf656ca-0da6-4a6c-80ce-6858d5b9d94f",
            "title": "Dark Matter"
          },
          "position": 8
        },
        {
          "number": "9",
          "artist-credit": [
            {
              "joinphrase": "",
              "name": "Teho",
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "name": "Teho",
                "country": "FR",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "sort-name": "Teho"
              }
            }
          ],
          "id": "330fd24e-84af-4f1a-8760-5506f3348f7b",
          "length": 388973,
          "title": "Mecha",
          "position": 9,
          "recording": {
            "artist-credit": [
              {
                "name": "Teho",
                "joinphrase": "",
                "artist": {
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "sort-name": "Teho",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "country": "FR",
                  "name": "Teho"
                }
              }
            ],
            "first-release-date": "2022-03-25",
            "id": "fbb2c10f-5ccd-46d0-9543-8c6d48ddf444",
            "length": 388973,
            "title": "Mecha",
            "video": false,
            "disambiguation": ""
          }
        },
        {
          "id": "28729108-a7ef-4001-99ac-006ee7ee151d",
          "artist-credit": [
            {
              "name": "Teho",
              "joinphrase": "",
              "artist": {
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                "type": "Person",
                "name": "Teho",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "country": "FR",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "sort-name": "Teho"
              }
            }
          ],
          "number": "10",
          "length": 388973,
          "title": "Odyssey",
          "position": 10,
          "recording": {
            "first-release-date": "2022-03-25",
            "artist-credit": [
              {
                "artist": {
                  "name": "Teho",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "country": "FR",
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
                  "type": "Person"
                },
                "joinphrase": "",
                "name": "Teho"
              }
            ],
            "id": "f0c5a9b5-4931-4e38-9172-075183d3941c",
            "length": 388973,
            "title": "Odyssey",
            "video": false,
            "disambiguation": ""
          }
        },
        {
          "position": 11,
          "recording": {
            "disambiguation": "",
            "video": false,
            "title": "Back Home",
            "length": 264226,
            "id": "07011610-83b9-434a-bdee-2b5682a47177",
            "first-release-date": "2022-03-25",
            "artist-credit": [
              {
                "artist": {
                  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                  "sort-name": "Teho",
                  "country": "FR",
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "name": "Teho",
                  "type": "Person",
                  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
                },
                "name": "Teho",
                "joinphrase": " & "
              },
              {
                "artist": {
                  "type": "Person",
                  "disambiguation": "Harold Grand-Jacquot",
                  "id": "9c8465cb-7778-4f51-a483-186ae66e37de",
                  "sort-name": "Mashk",
                  "country": null,
                  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                  "name": "Mashk"
                },
                "name": "Mashk",
                "joinphrase": ""
              }
            ]
          },
          "title": "Back Home",
          "id": "9e73bafc-80d7-4d93-9dac-889f111757ca",
          "number": "11",
          "artist-credit": [
            {
              "artist": {
                "sort-name": "Teho",
                "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
                "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
                "country": "FR",
                "name": "Teho",
                "type": "Person",
                "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France."
              },
              "name": "Teho",
              "joinphrase": ""
            }
          ],
          "length": 264226
        }
      ],
      "id": "1afe846e-29eb-3464-a8dc-891e48ab8475",
      "format-id": "9712d52a-4509-3d4b-a1a2-67c88c643e31",
      "track-offset": 0,
      "track-count": 11,
      "title": ""
    }
  ],
  "title": "Infinity",
  "date": "2022-03-25",
  "packaging": null,
  "quality": "normal",
  "text-representation": {
    "script": null,
    "language": null
  },
  "id": "e73e0fa9-97d6-4c38-97a0-2d60240f3a32",
  "cover-art-archive": {
    "front": true,
    "count": 1,
    "darkened": false,
    "back": false,
    "artwork": true
  },
  "status-id": "4e304316-386d-3409-af2e-78857eec5cfe",
  "status": "Official",
  "barcode": null,
  "release-events": [
    {
      "area": {
        "type-id": null,
        "sort-name": "France",
        "id": "08310658-51eb-3801-80de-5a0739207115",
        "disambiguation": "",
        "name": "France",
        "iso-3166-1-codes": [
          "FR"
        ],
        "type": null
      },
      "date": "2022-03-25"
    }
  ],
  "asin": null,
  "country": "FR",
  "packaging-id": null,
  "artist-credit": [
    {
      "joinphrase": "",
      "name": "Teho",
      "artist": {
        "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
        "type": "Person",
        "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
        "country": "FR",
        "sort-name": "Teho",
        "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
        "name": "Teho"
      }
    }
  ]
}