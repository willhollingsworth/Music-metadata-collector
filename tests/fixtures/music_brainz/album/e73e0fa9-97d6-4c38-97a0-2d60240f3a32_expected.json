{
  "album_name": "Infinity",
  "album_id": "e73e0fa9-97d6-4c38-97a0-2d60240f3a32",
  "status": "Official",
  "artist_name": "Teho",
  "artist_id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
  "track_count": 11
}